vitrivr-engine-core/src/main/kotlin/org/vitrivr/engine/core/model/retrievable/attributes/ContentAuthorAttribute.kt

```diff
@@ -0,0 +1,39 @@
+package org.vitrivr.engine.core.model.retrievable.attributes
+
+import java.util.*
+import kotlin.collections.HashMap
+
+class ContentAuthorAttribute private constructor(
```

Comment:
**I have a very fundamental question:** Why do we need such a `ContentAuthorAttribute` that maps UUIDs to authors? Why don't we just attach the author property directly to the `ContentElement`? 

This would remove one level indirection and prevents inconsistencies, e.g., if an `Operator` removes content but forgets to update the `ContentAuthorAttribute`.

Reply:
I think it is a bad design choice to grow the content classes as we make the pipelines more complex. This complexity should be outsourced in seperate objects in my opinion, with the contents themselves only needing an id. Furthermore, an operator should never remove content if we want to have pipelines that support branching.

Reply:
Resolving this after in person discussion.